utility/BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+# TODO: Rename file to 'BUILD' post-Gradle. See https://github.com/oppia/oppia-android/issues/1532.
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+'''A kt_android_library() rule that builds utility module kotlin source files into a library.'''
+kt_android_library(
+    name = "utility_lib",
+    custom_package = "org.oppia.util",
+    srcs = glob(["src/main/java/org/oppia/util/**/*.kt"],
+                exclude = ["src/main/java/org/oppia/util/logging/firebase/*.kt"]),
+    resource_files = glob(["src/main/res/**/*.xml"]),
+    manifest = "src/main/AndroidManifest.xml",
+    deps = [
+        ":dagger",
+        "//model:model_lib",
+        "//firebase:crashlytics_lib",
+        "//firebase:crashlytics_deps",
+        artifact("com.google.firebase:firebase-analytics:17.4.4"),
+        artifact("com.google.firebase:firebase-crashlytics:17.1.1"),
+        artifact("org.jetbrains.kotlinx:kotlinx-coroutines-core"),
+        artifact("org.jetbrains.kotlinx:kotlinx-coroutines-android"),
+        artifact("org.jetbrains.kotlinx:kotlinx-coroutines-test"),
+        artifact("com.google.truth:truth"),
```

Comment:
Here and elsewhere: none of these test deps shouldn’t be present in the production version of the library.

Reply:
okay!